SimMuon/Configuration/python/SimMuon_EventContent_cff.py

```diff
@@ -49,6 +49,7 @@
 phase2_muon.toModify( SimMuonFEVTDEBUG, outputCommands = SimMuonFEVTDEBUG.outputCommands + ['keep *_simMuonME0PseudoDigis_*_*',
                                                                                             'keep *_simMuonME0PseudoReDigis_*_*',
                                                                                             'keep *_simMuonME0Digis_*_*',
-                                                                                            'keep *_simMuonME0PadDigis_*_*'] )
+                                                                                            'keep *_simMuonME0PadDigis_*_*',
+                                                                                            'keep *_simMuonME0PadDigiClusters_*_*'] )
```

Comment:
Here is a dump of the ME0 digi collections in the digi ROOT file from the test (10 events) 27434.0_TTbar_14TeV+TTbar_14TeV_TuneCUETP8M1_2023D17_GenSimHLBeamSpotFull14+DigiFullTrigger_2023D17+RecoFullGlobal_2023D17+HARVESTFullGlobal_2023D17/
<PRE>
-bash-4.1$ edmEventSize -a -v step2.root | grep ME0
GEMDigiSimLinkedmDetSetVector_simMuonME0Digis_ME0_HLT. 15272.7 5071.5
StripDigiSimLinkedmDetSetVector_simMuonME0Digis_ME0_HLT. 14496.9 4676.4
PSimHits_g4SimHits_MuonME0Hits_SIM. 952.3 563.5
ME0DetIdME0DigiPreRecoMuonDigiCollection_simMuonME0PseudoReDigis__HLT. 715.9 428
ME0DetIdME0DigiPreRecoMuonDigiCollection_simMuonME0PseudoDigis__HLT. 713.1 418.7
ME0DetIdME0DigiMuonDigiCollection_simMuonME0Digis__HLT. 534.3 325.1
ME0DetIdME0PadDigiMuonDigiCollection_simMuonME0PadDigis__HLT. 514.9 309.4
ME0DetIdintMuonDigiCollection_simMuonME0PseudoReDigis__HLT. 471.7 304.4
ME0DetIdME0PadDigiClusterMuonDigiCollection_simMuonME0PadDigiClusters__HLT. 319.7 302.8
</PRE>

Compare with GEM
<PRE>
-bash-4.1$ edmEventSize -a -v step2.root | grep GEM
GEMDigiSimLinkedmDetSetVector_simMuonME0Digis_ME0_HLT. 15272.7 5071.5
GEMDigiSimLinkedmDetSetVector_simMuonGEMDigis_GEM_HLT. 14689.7 4790.7
StripDigiSimLinkedmDetSetVector_simMuonGEMDigis_GEM_HLT. 14359.1 4604.3
PSimHits_g4SimHits_MuonGEMHits_SIM. 546.7 402.4
GEMDetIdGEMPadDigiClusterMuonDigiCollection_simMuonGEMPadDigiClusters__HLT. 429.1 328.5
GEMDetIdGEMPadDigiMuonDigiCollection_simMuonGEMPadDigis__HLT. 383.5 305.1
GEMDetIdGEMDigiMuonDigiCollection_simMuonGEMDigis__HLT. 378.9 297.6
GEMDetIdGEMCoPadDigiMuonDigiCollection_simCscTriggerPrimitiveDigis__HLT. 313.5 296.3
</PRE>